### PR TITLE
Fix edit multi add layer

### LINKED ISSUE #1803: 1.6.7rc4 - Problem with tree order
If we add or remove a layer from a layergroup, all order values are reseted... The values entered manually are not kept. I saw another issue talking about that, but not sure if it is the same problem, because it had to be fixed in 1.6.7.

### PR DESCRIPTION
**Pull request by sbrunner**
fix #1803

### CODE REVIEW

File: c2cgeoportal/forms.py [hunk 1]
```diff
@@ -264,7 +264,10 @@ def render_organisational_item(self, item, depth):
         return result
 
     def is_checked(self, item, final_item):
-        return self._is_checked(final_item.id)
+        return final_item.id in self.values
```
Comment (ger-benjamin) [inline c2cgeoportal/forms.py]: Minor but: With a method that is named "is"_somethings, I expect a boolean value. Now, this seems return a number (id)...

Comment (sbrunner) [inline c2cgeoportal/forms.py]: no it return a boolean :-)
`>>> 1 in [1, 2, 3] => True`
:-)

Comment (ger-benjamin) [inline c2cgeoportal/forms.py]: Yeah, forget that...

File: c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po [hunk 2]
```diff
@@ -369,23 +369,23 @@ msgstr "Mode lecture écriture"
...
 "WARNING! an error occured while trying to read the mapfile and recover the "
 "themes."
-msgstr "ATTENTION ! Une erreur s'est produite en essayant de lire le mapfile  et de récupérer les thèmes."
+msgstr "ATTENTION ! Une erreur s’est produite en essayant de lire le mapfile  et de récupérer les thèmes."
 
 #: ./c2cgeoportal/views/entry.py:1325
 #, c-format
 msgid "User '%s' has no registered email address."
-msgstr "L'utilisateur '%s' n'a pas d'adresse mail enregistré."
+msgstr "L’utilisateur '%s' n’a pas d’adresse mail enregistré."
```
Comment (asaunier) [inline c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po]: enregistré => enregistrée
mail => email

Comment (sbrunner) [inline c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po]: fixed :-)

File: c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po [hunk 3]
```diff
@@ -369,23 +369,23 @@ msgstr "Mode lecture écriture"
...
 msgid "title i18n"
-msgstr "Test de l'i18n"
+msgstr "Test de l’i18n"
 
 #: ./c2cgeoportal/views/entry.py:130
 msgid ""
 "WARNING! an error occured while trying to read the mapfile and recover the "
 "themes."
-msgstr "ATTENTION ! Une erreur s'est produite en essayant de lire le mapfile  et de récupérer les thèmes."
+msgstr "ATTENTION ! Une erreur s’est produite en essayant de lire le mapfile  et de récupérer les thèmes."
```
Comment (ger-benjamin) [inline c2cgeoportal/locale/fr/LC_MESSAGES/c2cgeoportal.po]: Not your fault but can you correct that:
> ...le mapfile__et de...

Dobble spaces between 'mapfile' and 'et'

**Comment by ger-benjamin** [comment]
Not tested but seems alright.

**Pull request merged**